app.js

```diff
@@ -0,0 +1,99 @@
+const School = require("./School.js")
+const express = require("express")
+const cors = require("cors")
+const app = express()
+const port = 7000
+//const usersRouter = require("./School.js")
+const bodyParser = require("body-parser")
+
+app.use(bodyParser.urlencoded({extended:false}))
+app.use(bodyParser.json())
+app.use(cors())
+
+//app.use("/school", usersRouter)
+//app.use("/class", usersRouter)
+
+let mySchool = new School()
+
+// App.post for creating a new class
+// const checkIfClassExists = (req, res, next) =>{
+//     if (req.bl)
+// }
+
+
+
+app.get('/class', (req, res)=>{
+    let classes = mySchool.classes
+    res.json({
+        classes
+    })
+})
+
+
+app.get('/class/student', (req, res)=>{
+    let students = Student.name
+    res.json({
+        class: mySchool.classes[name]
+    })
+})
+
+
+const addNewClass = (req,res)=>{
+    let name = req.body.name
+    let teacher = req.body.teacher
+    console.log(name)
+    //let mySchool = new School()
+    if(!mySchool.classes[name]){
+        let newClass = mySchool.addClass(name, teacher)
+        res.json({
```

Comment:
if on the frontend we set the api request to a variable, we would get this response as a value.